cmd/cloudinitexecute/cloudinitexecute.go

```diff
@@ -84,6 +85,24 @@ func ApplyConsole(cfg *rancherConfig.CloudConfig) {
 			continue
 		}
 
+		targetDir := mount[1]
+		if mount[3] == "bind" {
+			fileInfo, err := os.Stat(mount[0])
+			if err != nil {
+				log.Errorf("Unable to find %s: %v", mount[0], err)
+				continue
+			}
+			if mode := fileInfo.Mode(); !mode.IsDir() {
+				targetDir = filepath.Dir(mount[1])
+			}
+		}
+		if targetDir != "" {
```

Comment:
`util.Mount()` does a `MkdirAll` - though yes, its not `bind`-mounty

so many little things.